### ISSUE TITLE
Difference between HTML and PDF versions of revealjs presentation combining `r-stack` with `absolute`

### ISSUE THREAD

**Issue by dmkaplan2000** [quarto-dev/quarto-cli#10379]
### Bug description

With the minimal document provided below, when I print the resulting presentation to PDF using either `pagedown::chrome_print` or printing directly from chrome, the absolute text only appears on the last slide, whereas it appears on all slides in the HTML version of the presentation.

### Steps to reproduce

Render the document below and then execute `pagedown::chrome_print("test.html")`. Compare the HTML and PDF versions.

``````quarto
---
title: "test"
format: 
  revealjs:
    pdf-separate-fragments: true
---

### Change fishing practices

:::{.r-stack}

:::{.fragment .fade-in-then-out}
![](https://static.pexels.com/photos/45201/kitty-cat-kitten-pet-45201.jpeg){height="450px"}
:::

:::{.fragment .fade-in}
![](https://wallup.net/wp-content/uploads/2018/10/06/708179-kittens-kitten-cat-cats-baby-cute-s.jpg){height="450px"}
:::

:::

:::{.absolute left="20%" bottom="1%" width="60%"}
SOME TEXT
:::


``````

### Expected behavior

Absoluted text appears on all slides.

### Actual behavior

In PDF, absoluted text only appears on last slide...

### Your environment

RStudio 2024.04.2+764 "Chocolate Cosmos" Release (e4392fc9ddc21961fd1d0efd47484b43f07a4177, 2024-06-05) for Ubuntu Jammy
Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) rstudio/2024.04.2+764 Chrome/120.0.6099.291 Electron/28.3.1 Safari/537.36, Quarto 1.5.27 (/opt/quarto/bin/quarto)

R version 4.4.1 (2024-06-14)

Ubuntu 22.04.4 LTS

### Quarto check output

```bash
quarto check
Quarto 1.5.27
[✓] Checking versions of quarto binary dependencies...
      Pandoc version 3.1.11: OK
      Dart Sass version 1.70.0: OK
      Deno version 1.41.0: OK
[✓] Checking versions of quarto dependencies......OK
[✓] Checking Quarto installation......OK
      Version: 1.5.27
      Path: /opt/quarto/bin

[✓] Checking tools....................OK
      TinyTeX: (external install)
      Chromium: (not installed)

[✓] Checking LaTeX....................OK
      Using: TinyTex
      Path: /home/dmk/.TinyTeX/bin/x86_64-linux
      Version: 2024

[✓] Checking basic markdown render....OK

[✓] Checking Python 3 installation....OK
      Version: 3.9.18 (Conda)
      Path: /home/dmk/miniforge-pypy3/bin/python
      Jupyter: (None)

      Jupyter is not available in this Python installation.
      Install with conda install jupyter

[✓] Checking R installation...........OK
      Version: 4.4.1
      Path: /usr/lib/R
      LibPaths:
        - /home/dmk/R/x86_64-pc-linux-gnu-library/4.4
        - /usr/local/lib/R/site-library
        - /usr/lib/R/site-library
        - /usr/lib/R/library
      knitr: 1.48
      rmarkdown: 2.27

[✓] Checking Knitr engine render......OK


```

**Comment by cscheid** [comment]
We'd like to do better. But this kind of cross-format CSS is very hard to fix in general, so I don't think we'll be able to solve it ourselves anytime soon.

**Comment by cderv** [comment]
This could also be a revealjs issue directly. We don't do anything specific for print function of revealjs, as this is built-in and we use their. 

We could try tweak some parts, but this is still an upstream issue probably. 

I'll test this with revealjs update to v5 (https://github.com/quarto-dev/quarto-cli/issues/4795)

**Comment by dmkaplan2000** [comment]
OK, thanks for taking a look. I figured this might be the case, but since I have rarely seen many differences between PDF and HTML, I did a bug report. The reason this is important is that sometimes presentations in HTML aren't allowed, so having a PDF backup is important...

**Comment by pchr8** [comment]
+1 for differences in absolute elements in PDF version, specifically if columns etc. are involved. Likely a margin issue

If one uses absolute elements to highlight things over other pictures this is a semantic difference that is important, but I think if all pictures are positioned absolutely it works. I'll try a bug-report later with reproducible code but this is just a +1 for HTML/PDF differences

**Comment by cderv** [comment]
In fact this is some chrome settings. When printing it is advice to follow what we describe in the doc with the setting. 
https://quarto.org/docs/presentations/revealjs/presenting.html#print-to-pdf

Especially Margin set to None which solve  this.

**Comment by dmkaplan2000** [comment]
Thanks for this information. Note that I used `pagedown::chrome_print`, not chrome directly to print the PDF (in part because of ease, but in part because I have found in the past that `pagedown::chrome_print` has fewer formatting issues than printing directly from chrome for some reason). I imagine it is possible to configure `pagedown::chrome_print` to remove margins as you suggest, but I have not looked into it.

In any case, thanks for the assistance.
